packages/gluestick/src/config/webpack/__tests__/buildEntries.test.js

```diff
@@ -0,0 +1,58 @@
+jest.mock('../../../generator', () => jest.fn());
+jest.mock('entries.json', () => ({
+  '/': {
+    component: 'path/to/main/component',
+    routes: 'path/to/main/routes',
+    reducers: 'path/to/main/reducers',
+  },
+  '/home': {
+    component: 'path/to/home/component',
+    routes: 'path/to/home/routes',
+    reducers: 'path/to/home/reducers',
+  },
+}), { virtual: true });
+jest.mock('glob', () => ({
+  sync: jest.fn(() => ['path/to/main/entry.js']),
+}));
```

Comment:
You can extract all these mocks in some file and reuse it in other tests like `buildServerEntries.test.js`.